src/components/ha-relative-time.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+
+import relativeTime from "../common/datetime/relative_time";
+
+import type { HomeAssistant } from "../types";
+
+@customElement("ha-relative-time")
+class HaRelativeTime extends LitElement {
```

Comment:
This should be an `UpdatingElement`, as we don't need the render function. Change `updated` to `update`.